game_eggs/steamcmd_servers/squad/egg-squad-wine.json

```diff
@@ -0,0 +1,172 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-04-11T17:42:26+01:00",
+    "name": "Squad Wine",
+    "author": "parker@parkervcp.com",
+    "description": "Squad is a realism-based military tactical first-person shooter video game developed and published by Canadian indie developer Offworld Industries exclusively through the Steam distribution platform. It is a spiritual successor to the Project Reality modification for Battlefield 2.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_latest": "ghcr.io\/parkervcp\/yolks:wine_latest"
+    },
+    "file_denylist": [],
+    "startup": ".\/start.sh",
```

Comment:
You can't use SH files for startup, and is not required here at all. You can tail the log through the startup command

Reply:
> 5. [ x] You verify that the start command applied does not use a shell script

Agreed but still used 😆

Reply:
Fixed 🎉

Reply:
> > 5. [ x] You verify that the start command applied does not use a shell script
> 
> Agreed but still used 😆

no i changed but i like start,sh but its ok

Reply:
> > > 5. [ x] You verify that the start command applied does not use a shell script
> > 
> > 
> > Agreed but still used 😆
> 
> no i changed but i like start,sh but its ok

I understand the convenience of using a start.sh script, but I strongly recommend against it for security reasons.
- Editable shell scripts can be modified by users, potentially introducing malicious code, or breaking the server configuration unintentionally.
- Using the 'Startup Command' field directly gives us precise control over the server launch process and its permissions. This helps prevent accidental or intentional misuse.
- It's a best practice too or at least not a taboo since the server startup is designed to be manageable through a configuration fields. This ensures maintainability and consistency across servers.
- As a preference it's really okay but since this will be shared to the general public then I think this practice should be followed.